### PR TITLE
Added missing callProc call for browser.

### PR DESCRIPTION
**Pull request by xYouLL**
[callProc Wiki](https://wiki.rage.mp/index.php?title=Browser::callProc)

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,8 +1,28 @@
 interface Mp {
+	events: EventMpPool;
+
 	trigger(name: string, ...args: any[]): void;
 	invoke(name: string, ...args: any[]): void;
 }
 
+declare interface EventMpPool {
```
Comment (LeonardSSH) [inline index.d.ts:8]: Can we remove `declare` keyword? There is no reason to use `declare` here.

File: index.d.ts [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+	 * @returns Returns a promise
+	 *
+	 * @example
+	 * ```js
+	 * mp.events.callProc('test_proc', 'test');
+	 * ```
+	 *
+	 *
+	 */
+	callProc<PromiseType, ArgsType>(eventName: string, args: ArgsType): Promise<PromiseType>;
```
Comment (LeonardSSH) [inline index.d.ts:23]: Instead of `<PromiseType, ArgsType>`, can we use `<T, K>`?

File: index.d.ts (hunk 1 above)
Comment (LeonardSSH) [inline index.d.ts:8]: Can we remove `declare` keyword? There is no reason to use `declare` here.

**Review by LeonardSSH — CHANGES_REQUESTED** [review]

**Comment by xYouLL** [comment]
Done! Sorry didn't want to click on re-review.

**Comment by LeonardSSH** [comment]
LGTM!

**Pull request merged**